scripts/build.py

```diff
@@ -103,7 +103,9 @@
     'third_party/generated/js/third_party.min.js.map',
     'third_party/generated/webfonts/*',
     '*.bundle.js',
-    '*.bundle.js.map')
+    '*.bundle.js.map'
+    '*.woff',
```

Comment:
Hi @vojtechjelinek, yes seems like this is blocking.
I wonder why it still hashes the font-files after adding at-most specific details in `FILEPATHS_NOT_TO_RENAME`.
did i miss something?